src/reuse/comment.py

```diff
@@ -520,6 +520,8 @@ class XQueryCommentStyle(CommentStyle):
     ".cljc": LispCommentStyle,
     ".cljs": LispCommentStyle,
     ".cmake": PythonCommentStyle,  # TODO: Bracket comments not supported.
+    # Visual Studio Code workspace file (JSON with Comments based)
```

Comment:
Let's remove this comment. Otherwise LGTM. Thank you @benoit-rolandeau-act !

Reply:
Done